README.md

```diff
@@ -1,30 +1,30 @@
 # Mysite
 ## A project in Django (Python 3.x)
-**A simple blogging platform for the modern era.**
+**_A simple blogging platform for the modern era._**
 
-This web application generates a simple but capable blog using Django. The platform allows authors to create text posts using a dedicated admin portal, and allows logged in users to post comments through a simple comment form. 
+####This web application generates a simple but capable blog using Django. The platform allows authors to create text posts using a dedicated admin portal, and allows logged in users to post comments through a simple comment form. 
 
-[example site](https://mighty-reaches-82829.herokuapp.com/)
+[Example Website](https://mighty-reaches-82829.herokuapp.com/)
 
-Create a beautiful and powerful blog in under 4mb! 
+**_Create a beautiful and powerful blog in under 4mb!_**
 
 ### FAQ
-Q. What can it do?
-A. Create simple but beautiful text based blog posts with important features such as commenting and user credential generation.
+  Q. What can it do?
+  A. Create simple but beautiful text based blog posts with important features such as commenting and user credential generation.
 
-Q. Who is it for? 
-A. Anyone looking for a simple but capable solution for a text only blog such as quotes or philosophy.
+  Q. Who is it for? 
```

Comment:
Please add 2 spaces at the end of this line to properly render the answer on the next line.